libraries/engage/core/hooks/useSettings.js

```diff
@@ -0,0 +1,11 @@
+import { useConfig } from './useConfig';
+
+/**
+ * Retrieves the global theme settings.
+ * @returns {Object}
+ */
+export function useSettings() {
+  const { settings = {} } = useConfig();
```

Comment:
I think this helper is not really useful, when only global settings are returned.
It would be better, if it would consider the current context (current widget, page and global all together).
That way a developer would not care in which scope the setting is placed, he would just ask for settings and expect that to be automatically be correct, wherever the component is placed. Otherwise he would always need to check where the component is placed and somehow determine the settings from these three places.

Reply:
Again, we will not have an automatic merging of settings. An external developer will get the possibility to decide on whether he wants the global or the local config. Everything else does not make sense right now.

Reply:
I think this does not fit the requirements that we have right now and it violates the config principle. It also makes it harder for us and everyone supposed to use it.
We should talk about it.